scripts/debian/falco.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Falco: Container Native Runtime Security
+
+[Service]
+Type=simple
+User=root
+ExecStart=/usr/bin/falco --pidfile=/var/run/falco.pid
```

Comment:
This is missing the logic to do the modprobe/dkms - if we accept this PR in the next update Falco will not behave as it does now.

Reply:
I'll look at this later to recap my memory.

Reply:
Thank you!

Reply:
Correct me if I am wrong.

If I remember correctly, without the PR change, falco performs the modprobe/dkms at the package install and uninstall stage. Therefore, this PR does not break the behavior change.

Reply:
Almost correct, but there's more. The current scripts are also calling [modprobe](https://github.com/falcosecurity/falco/blob/master/scripts/debian/falco#L67)/[rmmod](https://github.com/falcosecurity/falco/blob/master/scripts/debian/falco#L99) on the kernel module when starting/stopping the service.

I guess @fntlnz was referring to that. And I agree we should keep the same behavior.

PS
IIRC, the Falco process now tries to `insmod` the kernel module if it does not find it running yet. However, that behavior was implemented as a fallback and we should not rely on that. Moreover, it won't `rmmod` the kernel module. 
Thus, the systemd's script should still responsible for handling the whole kernel module lifecycle.

Reply:
Address `/sbin/modprobe falco` and `/sbin/rmmod falco` in systemd service file.